.github/workflows/run-nala.yml

```diff
@@ -19,5 +19,10 @@ jobs:
           labels: ${{ join(github.event.pull_request.labels.*.name, ' ') }}
           branch: ${{ github.event.pull_request.head.ref }}
           repoName: ${{ github.repository }}
+          prUrl: ${{ github.event.pull_request.head.repo.html_url }}
+          prOrg: ${{ github.event.pull_request.head.repo.owner.login }}
+          prRepo: ${{ github.event.pull_request.head.repo.name }}
+          prBranch: ${{ github.event.pull_request.head.ref }}
```

Comment:
branch and PrBranch are the exact same value, they are duplicates.  Can we use the previous "branch" variable rather than the prBranch?

Reply:
@amauch-adobe : yeah, they are the same; I will update them later once all nala-test running and branch selection are as per expectations.